tensorboard/webapp/widgets/data_table/filter_dialog_component.scss

```diff
@@ -30,15 +30,49 @@ limitations under the License.
       'background'
     );
   }
+}
+
+.input-container {
+  margin-bottom: 8px;
+}
+
+.discrete-filters-container {
+  max-height: 300px;
+  overflow-y: auto;
+}
+
+.no-matches {
+  // 12px is the width of the checkbox so the text should match the
+  // indentation of the selectable filters.
+  padding: 8px 12px;
+}
 
-  .discrete-filters-area {
-    max-height: 100px;
-    overflow-y: auto;
+.filter-container {
+  padding-bottom: 8px;
+  margin-bottom: 8px;
+  border-bottom: 1px solid mat.get-color-from-palette($tb-foreground, border);
+
+  @include tb-dark-theme {
+    border-bottom-color: mat.get-color-from-palette(
+      $tb-dark-foreground,
+      border
+    );
   }
+}
+
+.range-input {
+  padding: 4px 8px;
+  width: 200px;
+}
+
+.discrete-value {
+  overflow: hidden;
 
-  .no-matches {
-    // 12px is the width of the checkbox so the text should match the
-    // indentation of the selectable filters.
-    padding: 8px 12px;
+  ::ng-deep {
```

Comment:
I assume this is to increase clickable area. Given that this isn't rendered in any visible way, could you leave a comment explaining?

Reply:
That's a good point. I added a comment and I also posted some screenshots of the inspector (hopefully this illustrates the change).